Dockerfiles/ccp-openshift-scan/scanning/container-capabilities.py

```diff
@@ -46,7 +46,7 @@ def check_args(cmd):
             if not found_sec_arg:
                 print("\nThis container uses privileged "
                       "security switches:")
-            print("\nINFO: {}\n\t{}".format(sec_arg, security_args[sec_arg]))
+            print("\nINFO: {0}\n\t{1}".format(sec_arg, security_args[sec_arg]))
```

Comment:
The python version available in CentOS6 image does not understanding `{} {}`, one needs to provide explicit ordering.

When I fixed a scanner, I tested the fixes and all other scanners with CentOS6 and CentOS7 images to be sure it works as expected, I found this issue. We didn't hit this issue in testing because the execution failed before code could reach until here, the execution failed at the very start for python's absolute path.